src/parseUtils.js

```diff
@@ -45,7 +45,7 @@ function parseBundle(bundlePath) {
         // As function name may be changed with `output.jsonpFunction` option we can't rely on it's default name.
         if (
           node.callee.type === 'Identifier' &&
-          args.length === 2 &&
+          (args.length === 2 || args.length === 3) &&
           isArgumentContainsChunkIds(args[0]) &&
           isArgumentArrayConcatContainingChunks(args[1])
```

Comment:
I don't see any reason why there would be anything different than chunk IDs in the third place. I think I tracked pretty well the code path in webpack that generates the output, and I didn't see any signs of other code being sent as the third argument.

It used to always even be just two args, back in webpack 1, but webpack 2 changed it to fix some bug with entry points not being assigned to globals with UMD or something.

I feel pretty safe just checking for the arguments length. If someone later has a bug with this, it should be quite easy to fix ;)